ir/inst_other.go

```diff
@@ -674,3 +674,48 @@ func (inst *InstCleanupPad) LLString() string {
 	}
 	return buf.String()
 }
+
+// ~~~ [ freeze ] ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+// InstFreeze is an LLVM IR freeze instruction.
+type InstFreeze struct {
+	// Name of local variable associated with the result.
+	LocalIdent
+
+	X value.Value
+
+	// extra.
+
+	// (optional) Metadata.
+	Metadata
+}
+
+// NewInstFreeze returns a new freeze instruction based on the given
+// typed-value
```

Comment:
```suggestion
// operand.
```

Once more, the comment was just fine as you wrote it. I update it only to keep the code base consistent, e.g. this is the same as the comment used for the unary instruction `ir.NewFNeg`.